core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1113,7 +1113,7 @@ public final Cursor getNewItemIdsCursor() {
     public final Cursor getFeedMediaUnknownSizeCursor() {
         final String query = "SELECT * "
                 + " FROM " + TABLE_NAME_FEED_MEDIA
-                + " WHERE " + KEY_SIZE + ">" + Integer.MIN_VALUE;
+                + " WHERE " + KEY_SIZE + "<= 0";
```

Comment:
Couldn't you just check for 0? size Integer.MIN_VALUE won't be processed, anyway